packages/manager/modules/pci/src/projects/project/storages/cold-archive/users/translations/Messages_fr_FR.json

```diff
@@ -15,5 +15,6 @@
   "pci_projects_project_storages_containers_users_import_success_message": "Le téléchargement du fichier JSON lié à l’utilisateur {{user}} a été effectué avec succès.",
   "pci_projects_project_storages_containers_users_import_error_message": "Une erreur est survenue lors du téléchargement du fichier JSON lié à l’utilisateur : {{ message }}.",
   "pci_projects_project_storages_containers_users_show_secret_key_success": "La clé secrète de votre utilisateur {{user}} est {{secret}}.",
+  "pci_projects_project_storages_containers_users_show_secret_key_error": "Une erreur est survenue lors de la récupération de la clé secrète de l'utilisateur {{ user }} : {{ message }}",
```

Comment:
spaces don't matter ?

Reply:
if you're referring to the spaces between the accolades and the variables, then yes. They are added only for more readability.